Header  header
bool curb_L_FOUND
geometry_msgs/Point Point_L1
geometry_msgs/Point Point_L2
bool curb_R_FOUND
geometry_msgs/Point Point_R1
geometry_msgs/Point Point_R2
